.github/workflows/add-incident.yml

```diff
@@ -0,0 +1,26 @@
+name: Add comment
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  add-comment:
+    if: github.event.label.name == 'incident'
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Add comment
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            @${{ github.event.issue.user.login }} update the `incident` metrics below. Include relevant links to the sources of your values e.g. commit, pull request, slack alert, etc.
+            | Metric  | Value |
+            | ------------- | ------------- |
+            | When was the bug introduced (when did the incident begin to impact users)? | _YYYY-MM-DD_  |
+            | When was the bug reported in GitHub? | _YYYY-MM-DD_  |
```

Comment:
This workflow fires on issues creation right? So isn't this date the date the issue was created? Or is there an issue somewhere else? Why do we need this field?

Reply:
Sometimes, we create issues on our repos to track incident metrics for projects that are not being hosted by medic. For example echis, In that case we can't auto fill.

Reply:
You mean this issue may be created long after the initial issue is filed in the 3rd party repo?

Reply:
Yes, it may not be created the same day. But, it definitely makes sense to set this as the issue created date. I'll make that change.